wrapper/index.html

```diff
@@ -1,20 +1,37 @@
 <!DOCTYPE html>
 <html lang="en" ng-app="wizehive">
+
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Generate Test Data Plugin</title>
   <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,800' rel='stylesheet' type='text/css'>
   <link href='https://fonts.googleapis.com/css?family=Ubuntu:400,500' rel='stylesheet' type='text/css'>
-  <link rel="stylesheet" type="text/css" href="https://platform.zenginehq.com/dest/vendors.min.css">
```

Comment:
can we leave these in until we pull in the CSS from somewhere else? Since this PR just addresses the JS

Reply:
oh, yeah, that's fine.